ds.go

```diff
@@ -107,7 +104,7 @@ func (slice ByName) Less(i, j int) bool {
 }
 
 // Setup create the table which stores credentials
-func Setup() (err error) {
+func Setup(Table *string) (err error) {
```

Comment:
Can you just change `Table` to `tableName` in all calls, this matches the normal golang convention.

Aside from that it looks fantastic!

Thanks again.